components/pluralrules/examples/plurals.rs

```diff
@@ -0,0 +1,52 @@
+// An example application which uses icu_pluralrules to construct the correct
+// sentence for English based on the numerical value in Cardinal and Ordinal categories.
+use icu_data_provider::InvariantDataProvider;
+use icu_locale::LanguageIdentifier;
+use icu_pluralrules::{PluralCategory, PluralRuleType, PluralRules};
+
+static SILENT: bool = false;
+
+fn print(input: &str, value: Option<usize>) {
+    if SILENT {
+        return;
+    }
+    if let Some(value) = value {
+        println!("{}", input.replace("{}", &value.to_string()));
+    } else {
+        println!("{}", input);
+    }
+}
+
+fn main() {
+    let langid = LanguageIdentifier::default();
+    let values: &[usize] = &[0, 2, 25, 1, 3, 2, 4, 10, 7, 0];
+    let dtp = InvariantDataProvider;
+
+    {
+        print("\n====== Unread Emails (en) example ============", None);
```

Comment:
Suggestion: put these two examples (emails and elevator floors) into separate example files, one named `unread_emails.rs` and the other named `elevator_floors.rs`.  I think it'd be much nicer to get an error message like, "Caution: your PR increases the code size of unread_emails.rs by 12.3%!"